src/main/java/bio/terra/service/dataset/flight/create/CreateDatasetInitializeProjectStep.java

```diff
@@ -29,13 +29,19 @@ public StepResult doStep(FlightContext context) throws InterruptedException {
     BillingProfileModel profileModel =
         workingMap.get(ProfileMapKeys.PROFILE_MODEL, BillingProfileModel.class);
     String projectId = workingMap.get(DatasetWorkingMapKeys.GOOGLE_PROJECT_ID, String.class);
+    CloudPlatformWrapper platformWrapper =
+        CloudPlatformWrapper.of(datasetRequestModel.getCloudPlatform());
     GoogleRegion region =
-        CloudPlatformWrapper.of(datasetRequestModel.getCloudPlatform())
-            .getGoogleRegionFromDatasetRequestModel(datasetRequestModel);
+        platformWrapper.getGoogleRegionFromDatasetRequestModel(datasetRequestModel);
+    Boolean isAzure = platformWrapper.isAzure();
```

Comment:
unboxed types are better if you know `null` doesn't occur
```suggestion
    boolean isAzure = platformWrapper.isAzure();
```